bug-1669364-pref-verify-changing-location-service-providers-still-release-82-83.toml

```
@@ -1,12 +1,17 @@
 [metrics]
-weekly = ["geolocation_accuracy"]
-overall = ["geolocation_accuracy"]
+weekly = ["geolocation_accuracy_difference"]
+overall = ["geolocation_accuracy_difference"]
 
 
-[metrics.geolocation_accuracy]
+[metrics.geolocation_accuracy_difference]
 select_expression = "{{agg_histogram_mean('payload.histograms.geolocation_accuracy_exponential')}}"
 data_source = "main"
-[metrics.geolocation_accuracy.statistics.bootstrap_mean]
+[metrics.geolocation_accuracy_difference.statistics.bootstrap_mean]
 pre_treatments = ["remove_nulls"]
-[metrics.geolocation_accuracy.statistics.deciles]
+[metrics.geolocation_accuracy_difference.statistics.deciles]
 pre_treatments = ["remove_nulls"]
+
+[metrics.country]
+select_expression = "{{mozfun.stats.mode_last('country')}}"
```

```suggestion
select_expression = "mozfun.stats.mode_last(ARRAY_AGG(country))"
```

This isn't a Python function so we don't need Jinja to template anything, and since it's a SQL function it takes column names unquoted.

looking at the docs I'm pretty sure mode_last accepts an array and it doesn't perform aggregation on its own, so let's toss in an ARRAY_AGG to bundle it up for us.